app/src/main/java/com/example/concordia_campus_guide/Fragments/InfoCardFragment/InfoCardFragment.java

```diff
@@ -73,8 +78,9 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
     public void onActivityCreated(@Nullable Bundle savedInstanceState){
         super.onActivityCreated(savedInstanceState);
         mViewModel = ViewModelProviders.of(this).get(com.example.concordia_campus_guide.Fragments.InfoCardFragment.InfoCardFragmentViewModel.class);
-        mViewModel.setBuilding(ApplicationState.getInstance(getContext()).getBuildings().getBuilding(this.buildingCode));
+        mViewModel.setBuilding(this.buildingCode);
```

Comment:
i like the fact you were able to git rid of the context usage 👍  good job here!

Reply:
:D